### ISSUE TITLE
Ensure device is removed from running Zino state when a router is removed from `polldevs.cf`

### ISSUE THREAD

**Issue by lunkwill42** [Uninett/zino#142]

**Comment by lunkwill42** [comment]
I think this may already be implemented, but we should check.

**Comment by johannaengland** [comment]
Yes, this is already take care of when loading the pollfile: https://github.com/Uninett/zino/blob/41a87dcd86fbfdde26c063b267001f11a2ef5543/src/zino/scheduler.py#L63-L64

**Comment by johannaengland** [comment]
Actually, now that I think about it further, we do remove the device from `state.polldevs`, but we still keep a lot of information in `state.state`, specifically `state.state.devices` and `state.state.addresses` and potentially `state.state.flapping`
